commcare-cloud-bootstrap/commcare_cloud_bootstrap.py

```diff
@@ -124,27 +174,83 @@ def ask_aws_for_instances(env, aws_config, count):
             '--tag-specifications', 'ResourceType=instance,Tags=[{Key=env,Value=' + env + '}]',
         ]
         aws_response = subprocess.check_output(cmd_parts)
-        with open('ask_aws_for_instances.json', 'w') as f:
```

Comment:
Was this just broken before?

Reply:
Yeah, it was. This whole file is something I've just been working on and PRing occasionally, so I must have just left it broken.